### PR TITLE
log the work item id when notifying ado

### PR DESCRIPTION
**Pull request by chkeita**
## Summary of the Pull Request

Adding some logging to the bug filing

## PR Checklist
* [ ] Applies to work item: #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

File: src/api-service/__app__/onefuzzlib/notifications/ado.py [hunk 1]
```diff
@@ -209,15 +209,23 @@ def create_new(self) -> None:
                 self.project,
                 entry.id,
             )
+        return entry
 
-    def process(self) -> None:
+    def process(self, notification_info: str) -> None:
         seen = False
         for work_item in self.existing_work_items():
             self.update_existing(work_item)
```
Comment (mgreisen) [inline src/api-service/__app__/onefuzzlib/notifications/ado.py:217]: Is there a way to identify if the update fails?

Comment (chkeita) [reply src/api-service/__app__/onefuzzlib/notifications/ado.py]: an exception will be thrown if it fails

File: src/api-service/__app__/onefuzzlib/notifications/ado.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 # Licensed under the MIT License.
 
 import logging
-from typing import Iterator, List, Optional, Tuple, Union
+from typing import Any, Iterator, List, Optional, Tuple, Union
```
Comment (Porges) [inline src/api-service/__app__/onefuzzlib/notifications/ado.py:7]: unused?

File: src/api-service/__app__/onefuzzlib/notifications/ado.py [hunk 3]
```diff
@@ -175,6 +175,14 @@ def update_existing(self, item: WorkItem) -> None:
 
         if document:
             self.client.update_work_item(document, item.id, project=self.project)
+            logging.info(
+                "notify ado: updated work item" f" {item.id} - {notification_info}"
```
Comment (mgreisen) [inline src/api-service/__app__/onefuzzlib/notifications/ado.py:179]: How difficult is it to use a couple of customEvents for tracking when a work item is created and updated?

**Review by mgreisen — APPROVED** [review]

File: src/api-service/__app__/onefuzzlib/notifications/ado.py (hunk 3 above)
Comment (chkeita) [reply src/api-service/__app__/onefuzzlib/notifications/ado.py:179]: as discussed. i will add it in a different PR

**Pull request merged**